pkg/apis/alerting/v1/alerting.proto

```diff
@@ -431,6 +431,9 @@ message AlertEndpoint {
     EmailEndpoint email = 5;
     PagerDutyEndpoint pagerDuty = 6;
   }
+
+  // the following are read only
+  string id = 7;
```

Comment:
this seems weird

Reply:
*should be considered read only by any consumer of the API